src/utils/package-manager.ts

```diff
@@ -155,7 +201,7 @@ export const NPM: PackageManager = {
   },
 };
 
-export const packageManagers = [BUN, YARN_V1, YARN_V2, PNPM, NPM];
+export const packageManagers = [BUN, DENO, YARN_V1, YARN_V2, PNPM, NPM];
```

Comment:
Can you maybe change the ordering here. We just discussed this internally and think that ordering by popularity would make more sense here. 

```suggestion
export const packageManagers = [NPM, YARN_V1, YARN_V2, PNPM, BUN, DENO];
```